sdk/src/types/block/address/anchor.rs

```diff
@@ -40,6 +40,8 @@ impl AnchorAddress {
     }
 }
 
+impl StorageScore for AnchorAddress {}
```

Comment:
So this is just makes it 0?

Reply:
Yes

Reply:
Bit weird to even have it then but okay, guess we have it for consistency then